test/membrane/rtp/stream_receive_bin_test.exs

```diff
@@ -102,4 +105,84 @@ defmodule Membrane.RTP.StreamReceiveBinTest do
     assert_end_of_stream(pipeline, :sink)
     Testing.Pipeline.terminate(pipeline, blocking?: true)
   end
+
+  defmodule NoopSource do
```

Comment:
Sounds like something that should be achievable with TestingSource :thinking:

Reply:
I thought that as well, the trouble is that it must send data and if passed an empty list, it sends EoS right away.
Maybe if we had `Testing.PushSource` it would be helpful?

Reply:
I see, I didn't think about EoS. I would keep it